collectors/COLLECTORS.md

```diff
@@ -501,6 +501,40 @@ the `go.d.plugin`.
 These collectors are developed and maintained by third parties and, unlike the other collectors, are not installed by
 default. To use a third-party collector, visit their GitHub/documentation page and follow their installation procedures.
 
+<details>
+<summary>Typical third party Python collector installation instructions</summary>
+<br>
+In general the below steps should be sufficient to use a third party collector.
+
+1. Download collector code file into folder expected by Netdata.
+2. Download default collector configuration file into folder expected by Netdata.
+3. Edit configuration file from step 2 if required.
+4. Enable collector.
+5. [Restart Netdata](/docs/configure/start-stop-restart.md) 
+
+For example below are the steps to enable the [Python ClickHouse collector](https://github.com/netdata/community/tree/main/collectors/python.d.plugin/clickhouse).
+
+```bash
+# download python collector script to /usr/libexec/netdata/python.d/
+sudo wget https://raw.githubusercontent.com/netdata/community/main/collectors/python.d.plugin/clickhouse/clickhouse.chart.py -O /usr/libexec/netdata/python.d/clickhouse.chart.py
```

Comment:
For all `sudo` commands, I suggest to add `$ ` in front them, because they are command that we run on terminal.